acl_loader/main.py

```diff
@@ -718,21 +718,29 @@ def show_session(self, session_name):
         :param session_name: Optional. Mirror session name. Filter sessions by specified name.
         :return:
         """
-        header = ("Name", "Status", "SRC IP", "DST IP", "GRE", "DSCP", "TTL", "Queue", "Policer", "Monitor Port")
+        erspan_header = ("Name", "Status", "SRC IP", "DST IP", "GRE", "DSCP", "TTL", "Queue",
+                            "Policer", "Monitor Port", "SRC Port", "Direction")
+        span_header = ("Name", "Status", "DST Port", "SRC Port", "Direction", "Queue", "Policer")
 
-        data = []
+        erspan_data = []
+        span_data = []
         for key, val in self.get_sessions_db_info().iteritems():
             if session_name and key != session_name:
                 continue
-            # For multi-mpu platform status and monitor port will be dict()
-            # of 'asic-x':value
-            data.append([key, val["status"], val["src_ip"], val["dst_ip"],
-                         val.get("gre_type", ""), val.get("dscp", ""),
-                         val.get("ttl", ""), val.get("queue", ""), val.get("policer", ""),
-                         val.get("monitor_port", "")])
 
-        print(tabulate.tabulate(data, headers=header, tablefmt="simple", missingval=""))
+            if val.get("type") == "SPAN":
```

Comment:
What is the need for policer and queue for SPAN sessions?

Reply:
same as above. Extending it to SPAN also.